ext/misc/prefixes.c

```diff
@@ -204,7 +204,7 @@ static int prefixesBestIndex(
   ** (original_string) and use it if at all possible */
   int i;
   const struct sqlite3_index_constraint *p;
-
+  
```

Comment:
@stelkasouridis : please revert to committed version of this file. We don't want to commit changes that just include whitespace changes.